### PR TITLE
Main content refactor (figures)

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

File: src/components/pages/article/article-page.tsx [hunk 1]
```diff
@@ -108,14 +108,32 @@ export const ArticlePage = (props: { metaData: ArticlePageProps, abstract: Conte
           </div>
         </Tab>
         <Tab label="Figures and data">
-          <Heading id="figures" headingLevel={2} content="Figures and data" />
-          <ArticleContent content={getFigures(props.content)} />
+          <JumpToMenu headings={[
```
Comment (scottaubrey) [inline src/components/pages/article/article-page.tsx:111]: The JumpMenu in the page component need to be built with the headings from the content before we merge this.

**Comment by will-byrne** [comment]
This doesnt run for me, if you do a `docker-compose up` in the client it complains that:
```
./src/components/atoms/descriptors/descriptors.module.scss:16:1
Syntax error: Selector "img" is not pure (pure selectors must contain at least one local class or id)

  14 | }
  15 | 
> 16 | img {
     |     ^
  17 |   max-height: 100%;
  18 |   max-width: 100%;
```

**Review by scottaubrey — CHANGES_REQUESTED** [review]

**Comment by discodavey** [comment]
> 

This passed all linting and it should accept this

**Comment by scottaubrey** [comment]
Sorry Dave, I'm still getting the issue @will-byrne mentioned
Screenshot 2022-10-18 at 12 14 30 pm

**Comment by discodavey** [comment]
I'll take it out but not liking that we are being told we cannot have this as it is more of an opinion.

There will be other selectors that will need to be put in after Thursday so will need to allow them in the stylesheets

**Comment by scottaubrey** [comment]
Same deal with bare figure
Screenshot 2022-10-18 at 2 28 11 pm

For reference, this isn't linting failures. The style can't be built for production - it's a hard failure. And it's a failure because the selector is part of a component, but isn't scoped to the component. Like a selector lacking BEM-style selector in a sass file alongside a component, just enforced by the build-stage because it has no name to attach it to.

**Review by scottaubrey — APPROVED** [review]

**Pull request merged**